lib/view_model/advanced_privacy_settings_view_model.dart

```diff
@@ -78,9 +78,9 @@ abstract class AdvancedPrivacySettingsViewModelBase with Store {
         WalletType.ethereum,
         WalletType.polygon,
         WalletType.tron,
-        if (isRestore) WalletType.monero,
-        if (isRestore) WalletType.wownero,
-        if (isRestore) WalletType.zano,
+        WalletType.monero,
+        WalletType.wownero,
+        if (!isRestore) WalletType.zano,
```

Comment:
it should be in restore, but not in create
```suggestion
        if (isRestore) WalletType.zano,
```